docs/infra/api-markdown-documenter/render-api-documentation.mjs

```diff
@@ -92,7 +92,7 @@ export async function renderApiDocumentation(inputDir, outputDir, uriRootDir, ap
 		createDefaultLayout: layoutContent,
 		getAlertsForItem: (apiItem) => {
 			const alerts = [];
-			if (ApiItemUtilities.ancestryHasModifierTag(apiItem, "@system")) {
+			if (ApiItemUtilities.hasModifierTag(apiItem, "@system")) {
 				alerts.push("System");
```

Comment:
What do you think about using constants for these tag names?

Reply:
Personally, I think the code reads more easily with them as inline literals, especially since they're each only used in one place AFAIK. I don't feel too strongly about this, but I think I will at least consider this beyond the scope of the PR.